extensions/wrapper/client/src/test/java/de/sovity/edc/client/AssetApiServiceTest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ *  Copyright (c) 2023 sovity GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       sovity GmbH - initial API and implementation
+ *
+ */
+package de.sovity.edc.client;
+
+
+import org.eclipse.edc.connector.spi.asset.AssetService;
+import org.eclipse.edc.jsonld.spi.JsonLd;
+import org.eclipse.edc.junit.annotations.ApiTest;
+import org.eclipse.edc.junit.extensions.EdcExtension;
+import org.eclipse.edc.spi.protocol.ProtocolWebhook;
+import org.eclipse.edc.spi.types.domain.DataAddress;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.text.ParseException;
+import java.util.Map;
+
+import static de.sovity.edc.client.AssetTestUtils.createAsset;
+import static de.sovity.edc.client.AssetTestUtils.getDataAddress;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+
+@ApiTest
+@ExtendWith(EdcExtension.class)
+public class AssetApiServiceTest {
+
+
+    private static final String DATA_SINK = "http://my-data-sink/api/stuff";
+    private static final String COUNTER_PARTY_ADDRESS = "http://some-other-connector/api/v1/ids/data";
+
+    @BeforeEach
+    void setUp(EdcExtension extension) {
+        extension.registerServiceMock(ProtocolWebhook.class, mock(ProtocolWebhook.class));
+        extension.registerServiceMock(JsonLd.class, mock(JsonLd.class));
+        extension.setConfiguration(TestUtils.createConfiguration(Map.of()));
```

Comment:
Could we add a helper method TestUtils.setupExtension(Map.of()) so we don't have to keep adding these two lines to tests? (DRY + WET)